### ISSUE TITLE
warn again when internals are overriden

### ISSUE THREAD

**Issue by porres** [pure-data/pure-data#2414]
I remember Pd used to warn when an external did override an internal.

Currently, if you load [abs~] from zexy, for instance, and it overrides vanilla's [abs~], the only way for you to know is if you open its help file, which may also let people really confused.

a parallel issue might be to allow a configuration where this cannot happen (I don't care about that now, though)

**Comment by Spacechild1** [comment]
> I remember Pd used to warn when an external did override an internal.

Yes, that would be important! I can confirm that we don't get that warning anymore. Don't know what happened...

**Comment by umlaeute** [comment]
you still get the warnings, but only if Pd is started with `-verbose`:
```
warning: class 'abs~' overwritten; old one renamed 'abs~_aliased'
```

**Comment by porres** [comment]
I forgot, can we put it back in level 2 warnings? Feels important
